src/pages/home/report/ReportActionCompose.js

```diff
@@ -382,11 +385,19 @@ class ReportActionCompose extends React.Component {
      * @param {Boolean} shouldDebounceSaveComment
      */
     updateComment(comment, shouldDebounceSaveComment) {
-        const newComment = EmojiUtils.replaceEmojis(comment);
-        this.setState({
-            isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
-            value: newComment,
-        });
+        const emojiReplaceResults = EmojiUtils.replaceEmojis(comment);
+        const newComment = emojiReplaceResults.newText;
+
+        const isCommentEmpty = Boolean(newComment.match(CONST.REGEX.IS_COMMENT_EMPTY));
+        if (this.state.isCommentEmpty !== isCommentEmpty) {
```

Comment:
I think we can remove this check now as this is Pure Component.

Reply:
good catch, updated!